### ISSUE TITLE
Intel iGPU not showing.

### ISSUE THREAD

**Issue by bvalat** [ArcadeRenegade/SidebarDiagnostics#422]
**2. Your Operating System**
WIN 11

**3. App Version**
3.6.3

**4. Any Relevant Settings**
Default

**5. Steps to Reproduce**
Intel iGPU is not showed as GPU in the monitor settings.
It is seen and displayed in libre hardware monitor as "Intel(R) UHD Graphics" (joined ScreenShot)
![iGPU](https://user-images.githubusercontent.com/45319952/222125871-ef99b126-bde7-4754-bc73-1b5246d1b899.jpg)

Maybe is there just a missing mapping in your code?

**Comment by bvalat** [comment]
Maybe it is only missing GpuIntel in Monitoring.cs ?
return new HardwareType[2] { HardwareType.GpuNvidia, HardwareType.GpuAmd };
return new HardwareType[2] { HardwareType.GpuNvidia, HardwareType.GpuAmd, HardwareType.GpuIntel  };

**Comment by ArcadeRenegade** [comment]
Good catch can you test it out for me and make a PR? I don't have an Intel GPU to test.

**Comment by bvalat** [comment]
The problem is that i'm not a Dev.
But i managed to install Visul Studio and made the modification (Had to put a 3 instead of 2 since it seems to be the array size)
return new HardwareType[3] { HardwareType.GpuNvidia, HardwareType.GpuAmd, HardwareType.GpuIntel };
I run the project and voilà  : 
![image](https://user-images.githubusercontent.com/45319952/225782045-81c8825e-3c12-4a52-bc66-12283b2319ad.png)
VRAM seems to be stuck at 0. Since it is shared Memory, i think it is another metric. 
In Libre Hardware Monitor : 
![image](https://user-images.githubusercontent.com/45319952/225783427-ca7184ca-16ef-487f-b827-b6fa91d842e2.png)
Not shure how to change VRAMLoad to this data... Any hint for me?
I will try to make a pull request but i've never used github.

**Comment by bvalat** [comment]
And here is my first pull request ever :-)

**Comment by ArcadeRenegade** [comment]
Congratulations you are a developer now ☺️

**Comment by ArcadeRenegade** [comment]
Is your Intel GPU showing with your change?

**Comment by bvalat** [comment]
Yes 
Here is a N3450:
![image](https://user-images.githubusercontent.com/45319952/226130428-55a50a50-f55c-4dd5-be3e-85d1c6758844.png)
And a N5105: 
![image](https://user-images.githubusercontent.com/45319952/226131091-2d4617fc-8382-4ddf-9360-f1731291ede9.png)
And a core i7 with Nvidia 1060 MaxQ Laptop : 
![image](https://user-images.githubusercontent.com/45319952/226131394-36cf8ea1-9ff1-4a92-8420-c46996d5c8f2.png)
